manifests/init.pp

```diff
@@ -61,6 +61,9 @@
   $use_tls                  = false,
   $ca_file                  = undef,
   $permitted_peer           = undef,
+  $umask                    = '0000',
```

Comment:
All parameters must be validated and documented in the README.

Here's an example for validating these 4 digit octals

https://github.com/ghoneycutt/puppet-module-inittab/blob/master/manifests/init.pp#L21-22